sklearn/metrics/_regression.py

```diff
@@ -286,10 +286,12 @@ def mean_pinball_loss(
 def mean_absolute_percentage_error(
     y_true, y_pred, sample_weight=None, multioutput="uniform_average"
 ):
-    """Mean absolute percentage error regression loss.
+    """Mean absolute percentage error (MAPE) regression loss.
 
-    Note here that we do not represent the output as a percentage in range
-    [0, 100]. Instead, we represent it in range [0, 1/eps]. Read more in the
+    Note here that the output is not a percentage in the range [0, 100] and a value of 100
+    does not mean 100% but 1e2. Furthermore, the output can be arbitrarily high when `y_true` is small
+    (which is specific to the metric) or when `abs(y_true - y_pred)` is
+    large (which is common for most regression metrics). Read more in the
```

Comment:
Could you reformat that part to a line length of 79 characters? I think, this is still our default for examples. @thomasjpfan and @glemaitre might know better.

Reply:
I didn't find an automated way of doing it (`black .` doesn't seem to work for that) so I did it manually, hope it's fine.

Reply:
Yes, that's totally fine.